components/Deck/InfoPanel/PresentationsPanel.js

```diff
@@ -159,16 +159,16 @@ class PresentationsPanel extends React.Component {
         else
             toReturn = <div ref="presentationPanel">
                 <div className="ui basic segment" style={panelDIVStyles}>
+                    <h4 className="ui  header">Live Sessions</h4>
                     <Grid columns={2}>
                       <Grid.Column width={4}>
-                        <Popup trigger={<Button icon onClick={this.openChooseASessionModal.bind(this)} aria-label="Create a live session, Beta"><Icon name="record" size="large"/></Button>} content='Create a live session (beta)' />
+                        <Popup trigger={<Button icon onClick={this.openChooseASessionModal.bind(this)} aria-label="Create a live session"><Icon name="record" size="large"/></Button>} content='Create a live session (beta)' />
```

Comment:
why remove the beta from the aria label only, and not the popup text?

Reply:
removed beta from tooltip